src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -0,0 +1,372 @@
+import {
+  Component,
+  Host,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  Watch,
+  State,
+} from '@stencil/core'
+import { getClassNames } from 'src/liquid/utils/getClassNames'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ * @part arrow - all arrow items (`ld-button` elements)
+ * @part dots - list-items containing dots
+ * @part end - arrow to jump to the last item (`ld-button` element)
+ * @part item - all pagination items containing a number (`ld-button` elements)
+ * @part items - list containing all slidable items and the marker
+ * @part list-wrapper - list-item containing the `ul` element with slidable items
+ * @part marker - marker highlighting the selected item
+ * @part next - arrow to go to the next item (`ld-button` element)
+ * @part prev - arrow to go to the previous item (`ld-button` element)
+ * @part start - arrow to jump to the first item (`ld-button` element)
+ * @part sticky - all sticky items (`ld-button` elements)
+ * @part wrapper - list containing all pagination items
+ */
+@Component({
+  assetsDirs: ['assets'],
+  tag: 'ld-pagination',
+  styleUrl: 'ld-pagination.css',
+  shadow: true,
+})
+export class LdPagination {
+  @Element() element: HTMLElement
+
+  /** Label to communicate the type of an item. */
+  @Prop() itemLabel = 'Page'
+
+  /** The maximum number of items. */
```

Comment:
Why should a developer use this prop? Why should we offer it at all? The documentation should be more clear about it.

Reply:
Better now?